code-challenges.test.js

```diff
@@ -26,34 +26,149 @@ const numbersArray2 = [24, 27, 30, 33, 36]
 // a) Create a test with expect statements for each of the variables provided.
 
 const object1 = { number: 15 }
-// Expected output: "15 is divisible by three"
+
+describe("divisableByThree", () => {
+  it("returns if number is divisable by 3", () => {
+    expect(divisableByThree(15)).toEqual("15 is divisable by 3")
```

Comment:
Pay close attention to what you are passing into the function here vs what your given test variable is.  The test variable is an object, which will change how you approach the problem inside your function as you will need to access the value of the key number.